xmcl-runtime-api/src/util/launch.ts

```diff
@@ -153,6 +153,7 @@ export async function generateLaunchOptionsWithGlobal(
     side,
     resolution: inst.resolution ?? globalResolution,
     server: inst.server ?? undefined,
+    quickPlayMultiplayer: inst.server ? `${inst.server.host}${inst.server.port ? `:${inst.server.port}` : ''}` : undefined,
```

Comment:
Move this logic to LaunchServer.ts. You don't need to add an extra prop to here.